src/main/webapp/readfile.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="readfile.css">
     <script src="appscript.js"></script>
   </head>
-  <body onload="javascript:populateCustomerList()">
```

Comment:
Why remove this? Don't we still need to populate the customer list on load?

Reply:
This function doesn't work with the current build of the back-end.